Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme

```diff
@@ -102,6 +102,16 @@
             ReferencedContainer = "container:Client.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-com.apple.CoreData.ConcurrencyDebug 1"
+            isEnabled = "YES">
+         </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-com.apple.CoreData.SQLDebug 1"
```

Comment:
This shows us database mount location and SQL queries, it has 4 levels of verbosity. Turned off by default.